tools/MultiProjectValidator/AnalysisRules/DependencyMismatch/DependencyInfo.cs

```diff
@@ -9,7 +10,7 @@ public static DependencyInfo Create(ProjectContext context, LibraryDescription l
             return new DependencyInfo
             {
                 ProjectPath = context.ProjectFile.ProjectFilePath,
-                Version = library.Identity.Version.ToString(),
+                Version = library.Identity.Version?.ToString() ?? String.Empty,
```

Comment:
Makes sense. I disabled this test running against desktop earlier in the week when I ran into the same thing :smile: 
Seems like the test doesn't make much sense in that context anyways. 

Pointing it out to ensure it's not breaking any assumptions